elastic/integrations
# Add dashboards for EPR integration

## What does this PR do?

Add first dashboards and visualizations (lens) for the Elastic Package Registry (EPR) integration.

## Checklist


## How to test this PR locally

Testing is based on elastic-package command:

1. Boot up the Elastic stack using [elastic-package](https://github.com/elastic/elastic-package) from `elastic_package_registry` folder.
2. Boot up the EPR service by using the command: `elastic-package service up`
3. Get the container name to be used in configuration. It should be something similar to: `elastic-package-service_elastic_package_registry_1`
4. Add EPR integration in Kibana:
    - Set as hosts: `http://elastic-package-service_elastic_package_registry_1:9100`
    - Go to Discover and check in `metrics-*`. It can be filtered by `service.type: elastic_package_registry`
    - Go to Dashboards and open "[EPR] Overview"


## Related issues

- Relates https://github.com/elastic/package-registry/issues/797
- Requires https://github.com/elastic/integrations/pull/3671

## Screenshots

 WIP
